protocol_driver_grpc_async_callback.cc

```diff
@@ -316,4 +323,167 @@ void ProtocolDriverGrpcAsyncCallback::ShutdownServer() {
   }
 }
 
+using grpc::Status;
+
+class CallData {
```

Comment:
The name is a bit weird; CallBackHandler maybe ?

Reply:
The name comes from the grpc example code. RpcHandlerFSM might work.